### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by kbaran1998** [AntoniosBarotsis/qr-rs#10] [Ideas]
It should be possible to deploy the CLI tool to package repositories for Windows, macOS and popular Linux distributions.

**Comment by AntoniosBarotsis** [comment]
This would indeed be handy. 

I've only published on [chocolatey](https://chocolatey.org/) before and the experience was a little bit more convoluted than publishing to any of the platforms I am already targeting. I have no idea how publishing to brew works or for a myriad of different Linux distros but hopefully it's easier than it sounds.

Publishing to so many platforms also adds quite a bit of mental overhead so I might delay this for now or wait for help 😄
